libpod/runtime_ctr.go

```diff
@@ -460,3 +457,26 @@ func (r *Runtime) GetLatestContainer() (*Container, error) {
 	}
 	return ctrs[lastCreatedIndex], nil
 }
+
+// InitLabels allocates an new label to return to the caller
+func (r *Runtime) reserveLabels() error {
+	containers, err := r.state.AllContainers()
```

Comment:
This seems really expensive to do every time - do we really need to ReserveLabel every run, for all labels?

We might want to maintain this in the DB for perf reasons...

Reply:
I don't think so.  we could be smarter about only doing this if the containers have changed.  We should be checking the database at this time to make re read the list of containers, incase some one else used one.

Reply:
Is there an easy way to see if the database has changed since the last time we walked the list.

Reply:
We could add a separate bucket in the DB for a label->container ID map, which would let us do fast lookups to see if a given label was in use